resources/lang/en-US.yml

```diff
@@ -1792,4 +1792,6 @@ msg_err_cannot_join_nation_over_resident_limit: '&cYou cannot join this nation a
 msg_err_cannot_join_nation_over_town_limit: '&cYou cannot join this nation, it has reached the maximum number of allowed towns-per-nation: %s.'
 
 #Message shown confirming /tw set usedefaults
-confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'
\ No newline at end of file
+confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'confirmation_are_you_sure_you_want_to_reset_this_worlds_settings: 'Are you sure you want to reset all of this world''s settings to the ones which are set in the config?'
+#Message giving confirmation warning for trusttown
+confirmation_msg_trusttown_consequences: 'Are you sure you wish to trust all residents in this town? This may include future residents of this town if not revoked.'
```

Comment:
'You are about to give full plot permissions to an entire town. This means that the mayor of the town will be able to decide who gets permission to build and break in your town, by adding residents. If this town is open, then anyone can join and gain full access to your town. Are you sure you wish to trust all residents (including future residents,) in this town?'

Reply:
Have implemented this confirmation text in new commit.